lib/integrations/facebook/delivery_status.rb

```diff
@@ -1,32 +1,37 @@
 # frozen_string_literal: true
 
 class Integrations::Facebook::DeliveryStatus
-  def initialize(params)
-    @params = params
-  end
+  pattr_initialize [:params!]
 
   def perform
-    update_message_status
+    return if facebook_channel.blank?
+    return unless conversation
+
+    process_delivery_status if params.delivery_watermark
+    process_read_status if params.read_watermark
   end
 
   private
 
-  def sender_id
-    @params.sender['id']
+  def process_delivery_status
+    timestamp = Time.zone.at(params.delivery_watermark.to_i).to_datetime
```

Comment:
Can you confirm if this saves the time in UTC 

Cc: @sojan-official can you confirm if this is required

Reply:
I believe `Time.zone` defaults to UTC.  The payload is something like this,
![CleanShot 2023-11-16 at 13 08 49@2x](https://github.com/chatwoot/chatwoot/assets/12408980/70fc2981-849d-481a-b206-7e9d75287909)

Reply:
https://chat.openai.com/share/781e05cf-afab-488d-afb3-4885e7df3d82

Reply:
@sojan-official @scmmishra Fixed.